Week 4/Enigma-java/src/main/java/rotorEncryption.java

```diff
@@ -0,0 +1,60 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class rotorEncryption 
+{
+    
+
+    public static String testRotorEncrypt(String msg, List<String> rotors)
+    {
+        final String alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        String encryptedMsg = "";
```

Comment:
- [ ] Consider using a StringBuilder and methods such as append() for efficiency of code